include/io.tcc

```diff
@@ -2,24 +2,24 @@
 //! \details Get stress vector of vertex coordinates, and number of vertices
 //! \tparam Tdim dimension
 template <unsigned Tdim>
-void IO<Tdim>::write_material_points(
-    const std::vector<std::shared_ptr<Point<Tdim>>>& materialpoints) {
+void IO<Tdim>::write_point_coordinates(
+    const std::vector<std::shared_ptr<Point<Tdim>>>& points) {
```

Comment:
Should an IO class have the ability to modify `Point`? can we have this as a vector of coordinates?

Reply:
Which could be returned by the mesh class